### PR TITLE
Release note for 3.8.0 version

### PR DESCRIPTION
**Pull request by VincentRouvreau**

### CODE REVIEW

File: .github/next_release.md [hunk 1]
```diff
@@ -1,22 +1,40 @@
...
+
+- [Cover Complex](https://gudhi.inria.fr/python/latest/cover_complex_sklearn_user.html)
+     - New classes to compute Mapper, Graph Induced complex and Nerves with a scikit-learn like interface.
+
+- [Persistent cohomology](https://gudhi.inria.fr/doc/latest/group__persistent__cohomology.html)
+     - New linear-time `compute_persistence_of_function_on_line`, also available though `CubicalPersistence` in Python.
+
+- [Cubical complex](https://gudhi.inria.fr/doc/latest/group__cubical__complex.html)
+     - Add possibility to build a lower-star filtration from vertices instead of top-dimensional cubes.
+     - Faster persistence computation of 1d cubical complexes.
```
Comment (mglisse) [inline .github/next_release.md:20]: I would skip this one. The thing that really improves persistence of 1d cubical complexes is the new `compute_persistence_of_function_on_line` (that name is not pretty), but it isn't part of the C++ or Python cubical complex.

**Review by mglisse — COMMENTED** [review]
Looks ok otherwise, thanks.

File: .github/next_release.md [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
+     - Naming the arguments is now mandatory in CubicalComplex python interface.
```
[the remark this answers is not in the document]
Comment (VincentRouvreau) [reply .github/next_release.md]: Specified on [9e6dc35](https://github.com/GUDHI/gudhi-devel/pull/852/commits/9e6dc35044660e6d536a35023c5497ce4f580bbc)

File: .github/next_release.md [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
      - New `choose_n_farthest_points_metric` as a faster alternative of `choose_n_farthest_points`.
 
-- [Module](link)
-     - **...**
+- [SimplexTree](https://gudhi.inria.fr/python/latest/simplex_tree_ref.html)
+     - `SimplexTree` can now be used with `pickle`.
+
+- Installation
+     - CMake 3.8 is the new minimal standard to compile the library.
+     - Support for oneAPI TBB (instead of deprecated TBB) to take full advantage of multicore performance
```
[the remark this answers is not in the document]
Comment (VincentRouvreau) [reply .github/next_release.md]: Remove **full** on [9e6dc35](https://github.com/GUDHI/gudhi-devel/pull/852/commits/9e6dc35044660e6d536a35023c5497ce4f580bbc)

File: .github/next_release.md [hunk 4]
```diff
@@ -1,22 +1,40 @@
 We are pleased to announce the release 3.8.0 of the GUDHI library.
 
-As a major new feature, the GUDHI library now offers Perslay, a Tensorflow model for the representations module.
+As a major new feature, the GUDHI library now offers Perslay, a Tensorflow model for the representations module, scikit-learn like interfaces for Cover Complexes, a new function to compute persistence of a function on $\mathbb{R}$ and the possibility to build a Cubical Complex as a lower-star filtration from vertices.
```
[the remark this answers is not in the document]
Comment (VincentRouvreau) [reply .github/next_release.md]: Should be ok for github flavored markdown, but maybe not for the website. Modified on [9e6dc35](https://github.com/GUDHI/gudhi-devel/pull/852/commits/9e6dc35044660e6d536a35023c5497ce4f580bbc)

**Pull request merged**